monitor/tests/test_views.py

```
@@ -0,0 +1,284 @@
+        assert response.status_code == status.HTTP_200_OK
+        assert response.data['services'] == []
+        assert response.data['global_digest'] is None
+    
+    def test_health_summary_with_data(self):
+        """Test health summary with services and data."""
+        # Create services
+        service1 = baker.make(Service, name='API1', enabled=True)
+        service2 = baker.make(Service, name='API2', enabled=True)
+        
+        # Create health check results
+        result1 = baker.make(HealthCheckResult, service=service1, ok=True)
```

Ruff (0.12.2)

Local variable `result1` is assigned to but never used

Remove assignment to unused variable `result1`

(F841)